frontend/src/api/myCourses/getMyCourses.ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import { AxiosError } from 'axios';
+
+import { apiClientWrapper } from 'api/base';
+import { getUserIdCookie } from 'utils/helpers/getUserIdCookie';
+import { API } from 'constants/routes';
+import { REQUEST_ERRORS } from 'constants/authConstants';
+import { Course } from 'types/course';
+
+const useGetMyCourses = (): UseQueryResult<Array<Course>, AxiosError> =>
+  useQuery('myCourses', async () => {
+    let myCoursesResponse: Array<Course>;
+    const apiClient = apiClientWrapper();
+    const userId = getUserIdCookie();
+    try {
+      const response = await apiClient.get(`${API.getMyCourses}/${userId}`);
+      myCoursesResponse = response.data;
```

Comment:
you can use "const" here cause myCoursesResponse is never reassigned

Reply:
it is reassigned in line 17

Reply:
yeah, but before it's "undefined"